codebase-pipelines/buildspec-deploy.yml

```diff
@@ -75,92 +84,101 @@ phases:
 
       # Get service name
       - service_name=$(aws ecs list-services --cluster "${cluster}" | jq -r '.serviceArns[] | select(test("'${cluster}'-'${SERVICE}'($|-Service.*)"))' | cut -d '/' -f3)
-
-      # Update task definition
-      - task_definition=$(aws ecs describe-task-definition --task-definition "${task_family}")
-      - new_task_definition=$(echo ${task_definition} | jq '.taskDefinition | .containerDefinitions |= map(if .name == '\"${SERVICE}\"' then .image = '\"${image_uri}\"' else . end) | del(.taskDefinitionArn) | del(.revision) | del(.status) | del(.requiresAttributes) | del(.compatibilities) |  del(.registeredAt)  | del(.registeredBy)')
-      - new_task_info=$(aws ecs register-task-definition --cli-input-json "${new_task_definition}")
-      - new_revision=$(echo ${new_task_info} | jq '.taskDefinition.revision')
-
-      # Get desired task count
-      - count=0
-      - cd "${CODEBUILD_SRC_DIR}"
-
-      # If count is a range, get the first value otherwise get count
+      
+      # Run Copilot package command and clean the manifest output
+      - copilot svc package --env "${ENVIRONMENT}" --name "${SERVICE}" | yq .Metadata.Manifest | jq -r . | yq 'del(.image.location)' > new.yml
```

Comment:
When I run this manually in the demodjango repo it fails because jq can't parse the YAML. How come there is a mix of jq and yq in here?

I think you can achieve the same results using just yq.

Reply:
Actually, just removing the "jq -r ." from the command works locally for me.